x-pack/functionbeat/include/fields.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 func init() {
-	if err := asset.SetFields("functionbeat", "fields.yml", Asset); err != nil {
+	if err := asset.SetFields("functionbeat", "fields.yml", asset.LibbeatFieldsPri, Asset); err != nil {
```

Comment:
Is this `LibbeatFieldsPri` because it includes libbeat fields for the time being? Then when we separate out libbeat for each Beat this changes to `BeatFieldsPri`? Same question for some of the other Beats that have the same thing (except for Metricbeat).

Reply:
I adjusted this to always be `BeatFieldsPri` so in the future if we have Libbeat also in fields.go this should the prio here does not change, only the content.